locales/en-US/strings.properties

```diff
@@ -92,6 +92,9 @@ edit_topsites_unpin_button=Unpin this site
 edit_topsites_edit_button=Edit this site
 edit_topsites_dismiss_button=Dismiss this site
 
-# LOCALIZATION NOTE (pocket): This is shown in the PocketStories component.
-pocket_read_more=Read More
+# LOCALIZATION NOTE (pocket_read_more): This is shown below the Trending Stories
+# by Pocket row, followed by a list of trending topic links.
```

Comment:
The "Trending Stories by Pocket row" seems a bit specific and dependent on the other string. Maybe just
"This is shown at the bottom of the trending section and precedes a list of trending topic text links."